Package.swift

```diff
@@ -0,0 +1,10 @@
+import PackageDescription
+
+let package = Package(
+    name: "CrateIO",
+    dependencies: [
+        .Package(url: "https://github.com/VeniceX/HTTPServer.git", majorVersion: 0, minor: 4),
+        .Package(url: "https://github.com/Zewo/JSON.git", majorVersion: 0, minor: 4),
```

Comment:
Instead of using JSON directly, it's a better practice to use [ContentNegotiationMiddleware](https://github.com/zewo/contentnegotiationmiddleware) combined with [JSONMediaType](https://github.com/zewo/jsonmediatype).